sdk/storage/azure-storage-blob-cryptography/src/main/java/com/azure/storage/blob/specialized/cryptography/EncryptionVersion.java

```diff
@@ -20,5 +20,12 @@ public enum EncryptionVersion {
      * Version 2 of the client side encryption protocol.
      * Uses AES/GCM/NoPadding
      */
-    V2
+    V2,
+
+    /**
+     * Version 2.1 of the client side encryption protocol. Use this version when configuring
+     * authenticatedRegionDataLength for encryption.
+     * Uses AES/GCM/NoPadding
+     */
+    V2_1
```

Comment:
We should also make sure to update the changelog for this new encryption protocol and note in the previous authenticated region changelog that the protocol version needs to be set to `V2_1`